src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +123,24 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * <p>This method sends a message as if it came from this player.
```

Comment:
```java
     * <p>This method sends a message as if it came from this player.
     * To send a message to this player instead, see
     * {@link MessageReceiver#sendMessage(Text)} or
     * {@link ChatTypeMessageReceiver#sendMessage(ChatType, Text)}.</p>
     *
     * <p>Commands cannot be sent using this method. To send commands, use
     * {@link CommandManager#process(CommandSource, String)}.</p>
     *
     * <p>If text formatting is not supported in the implementation
     * it will be displayed as plain text.</p>
```